src/poliastro/twobody/orbit.py

```diff
@@ -216,12 +216,29 @@ def h_vec(self):
         h_vec = np.cross(self.r.to(u.km).value, self.v.to(u.km / u.s)) * u.km ** 2 / u.s
         return h_vec
 
+    @property
+    def h_mom(self):
+        """Angular momentum. """
```

Comment:
Notice that this is the _specific_ angular momentum. In mechanics, they call "specific" to the magnitudes without mass, like this case, see https://www.wikiwand.com/en/Specific_relative_angular_momentum